packages/components/slider/src/index.vue

```diff
@@ -282,20 +344,28 @@ export default defineComponent({
 
     return {
       ns,
+      t,
```

Comment:
**Suggestion:**  we don't really have to return this `t` since we didn't use it in the template.